unitTests/ui/ErrorReportTest.ts

```diff
@@ -140,6 +140,22 @@ export function ErrorReportTest() {
       );
     });
 
+    it('should display a different error message when the organization is paused', () => {
```

Comment:
is there any other potential situation the might happen with Coveo org that we need to handle as well? or we will fix it case by case